packages/shared-ux/page/solution_nav/src/solution_nav.tsx

```diff
@@ -174,6 +176,32 @@ export const SolutionNav: FC<SolutionNavProps> = ({
     );
   }, [children, headingID, isCustomSideNav, isHidden, items, rest]);
 
+  const navWidth = useMemo(() => {
+    if (isLargerBreakpoint) {
+      return isOpenOnDesktop ? `${FLYOUT_SIZE}px` : euiThemeVars.euiSizeXXL;
+    }
+    if (isMediumBreakpoint) {
+      return isSideNavOpenOnMobile || !canBeCollapsed
+        ? `${FLYOUT_SIZE}px`
+        : euiThemeVars.euiSizeXXL;
```

Comment:
Looks like there is no way (or no straightforward way) to connect these widths with the widths that are actually set for the sidebar. Maybe it is worth at least adding a comment in the CSS where these widths are set, that this logic might also need an update in case the CSS is updated

like here https://github.com/elastic/kibana/blob/51569b4c93fec61ad42a149d9899e9e93ed23ad9/packages/shared-ux/page/solution_nav/src/with_solution_nav.tsx#L77
or here https://github.com/elastic/kibana/blob/c39352f268474dce4bf2b5505891efa3d3a8e56c/packages/shared-ux/page/solution_nav/src/solution_nav.scss#L17

Reply:
Or maybe it is worth at least connecting them with css variable

Reply:
@Dosant I added variables for the sizes and added a comment about it also being set in the tsx file.

I don't think there is way to read the variables from the scss file in the tsx file at compile time without configuration changes. But if I'm wrong about that let me know.